recipes/heaptrack/meta.yaml

```
@@ -0,0 +1,68 @@
+
+{% set name = "heaptrack" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KDE/heaptrack/archive/v{{ version }}.tar.gz
+  sha256: f299a4846b80e607a412f439e17cddae13f0529701ffdb05eaa7ba878865717e
+  folder: source
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+  patches:
+    - stdc_format.patch
+
+build:
+  number: 0
+  skip: true  # [win]
```

`libunwind` and `elfutils` are only available for Linux and I think only the GUI supports macOS
```suggestion
  skip: true  # [not linux]
```